src/reference/asciidoc/ip.adoc

```diff
@@ -201,6 +205,8 @@ By default, reverse DNS lookups are done on inbound packets to convert IP addres
 In environments where DNS is not configured, this can cause delays.
 You can override this default behavior by setting the `lookup-host` attribute to `false`.
 
+Starting with version 5.3.3, you can add a `SocketCustomizer` bean to modify the `DatagramSocket` after it is created. It is called for the receiving socket and any sockets created for sending acks.
```

Comment:
Every sentence in the docs from a new line, please: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Probably we need to add a `What's New` entry as well.

I'll take care about it on cherry-pick.
Thanks